canonical-web-and-design/charmhub.io
# fix: beta store redirects

## Done
- manually init app instead of using `store_base` which doesnt read redirects properly
## How to QA
- make sure redirects defined in `redirects.yaml` work
- make sure forms work (no CSFR issues)
- click around, try to break stuff

## Testing
- [ ] No testing required (explain why): no new features

## Issue / Card
Fixes #

## Screenshots
